crypto/s2n_aead_cipher_aes_gcm.c

```diff
@@ -191,3 +191,36 @@ struct s2n_cipher s2n_aes256_gcm = {
     .set_decryption_key = s2n_aead_cipher_aes256_gcm_set_decryption_key,
     .destroy_key = s2n_aead_cipher_aes_gcm_destroy_key,
 };
+
+/* TLS 1.3 GCM ciphers */
+struct s2n_cipher s2n_tls13_aes128_gcm = {
+    .key_material_size = 16,
+    .type = S2N_AEAD,
+    .io.aead = {
+                .record_iv_size = 0, /* Implicit IV */
+                .fixed_iv_size = 12,
```

Comment:
comment/constant this value to be the Per-Record Nonce N_MIN for the AES AEAD algorithm
- https://tools.ietf.org/html/rfc8446#section-5.3 
- https://tools.ietf.org/html/rfc5116#section-4